working-groups/wg-tidb-distributions.md

```diff
@@ -0,0 +1,34 @@
+# TiDB Distributions Working Group
```

Comment:
please add this working group to the active working group list in `working-groups/README.md` file.

Reply:
Added